features/spec_files/frozen_string_literal.feature

```diff
@@ -0,0 +1,16 @@
+Feature: frozen string literal
+
+  Scenario: `.spec`
+    Given a file named "no_frozen_string_literal_spec.rb" with:
+      """ruby
+      RSpec.describe "something" do
+        it "does something" do
+          a = 'hello'
+          a << ' world'
+          expect(a).to eq('hello world')
+        end
+      end
+      """
+    When I run `rspec no_frozen_string_literal_spec.rb`
+    Then the examples should all pass
+
```

Comment:
I might have misperceived some of the comments in the other issues, but it looked like some people had concern if this would break existing specs. I added this one just to prove a test that would break with frozen string literal enabled still won't break without it.

That said, I don't think there's really much value in keeping it, so don't have a problem removing it.